src/main/java/de/zalando/aruha/nakadi/controller/EventStreamController.java

```diff
@@ -66,11 +56,11 @@ public EventStreamController(final TopicRepository topicRepository, final Object
     @RequestMapping(value = "/event-types/{name}/events", method = RequestMethod.GET)
     public StreamingResponseBody streamEvents(
             @PathVariable("name") final String eventTypeName,
-            @RequestParam(value = "batch_limit", required = false, defaultValue = "1") final int batchLimit,
-            @RequestParam(value = "stream_limit", required = false, defaultValue = "0") final int streamLimit,
-            @RequestParam(value = "batch_flush_timeout", required = false, defaultValue = "30") final int batchTimeout,
-            @RequestParam(value = "stream_timeout", required = false, defaultValue = "60") final int streamTimeout,
-            @RequestParam(value = "stream_keep_alive_limit", required = false, defaultValue = "0") final int streamKeepAliveLimit,
+            @RequestParam(value = "batch_limit", required = false) final Integer batchLimit,
+            @RequestParam(value = "stream_limit", required = false) final Integer streamLimit,
+            @RequestParam(value = "batch_flush_timeout", required = false) final Integer batchTimeout,
+            @RequestParam(value = "stream_timeout", required = false) final Integer streamTimeout,
+            @RequestParam(value = "stream_keep_alive_limit", required = false) final Integer streamKeepAliveLimit,
```

Comment:
Another convention I forgot to tell you: null handling.
In Nakadi we consider everything to be none-nullable, if there is something that can be null - it should be marked as `@Nullable`
I believe all these parameters are nullable.